aplt/runner.py

```diff
@@ -118,13 +118,25 @@ def connect(self, processor):
         self._connect_waiters.append(processor)
         connectWS(self._factory, contextFactory=self._factory_context)
 
-    def send_notification(self, processor, url, data, ttl, claims=None):
-        """Send out a notification to a url for a processor"""
+    def send_notification(self, processor, url, data, ttl, claims={}):
```

Comment:
definitely don't want the ol' [mutable default argument antipattern](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments)